content/blog/2021/09/2021-09-20-election-period-opened.adoc

```diff
@@ -0,0 +1,103 @@
+
+We already had two successful editions in a row. I want us to continue on that path.
+This is a tremendous opportunity for community members to influence the direction of the project for the next two years.
+To make this year’s election even better, we slightly modified the process by leveraging our new community platform aka link:https://community.jenkins.io[community.jenkins.io].
+
+To participate in the election, we ask every Jenkins community member to have an account on link:https://community.jenkins.io[community.jenkins.io].
+You can either reuse your Github account or create a new discourse account specific to link:https://community.jenkins.io[community.jenkins.io].
+The second requirement is to be able to showcase at least one contribution done before the first of September 2021.
+As mentioned on link:/participate/[jenkins.io.io/participate], they are many different ways to contribute to Jenkins and for many of them, it’s very difficult to measure.
+Therefore we’ll trust participants and will not require that they provide evidence of contribution as part of their voter registration. We reserve the right to ban the specific account from the election process if we identify abuse.
+The election works in three stages:
+
+. Identify voters and nominees
+. Voting period
+. Announce results
+
+## Voters 
+
+To invite participants to vote, we need a list of email’s addresses that we would share with the link:https://civs.cs.cornell.edu[Condorcet Internet Voting Service].
+Therefore we ask every community members who match the requirement to join the group link:https://community.jenkins.io/g/election-voter[election-voter] on link:https://community.jenkins.io[community.jenkins.io] .
+The group will be open for joining during the registration period after we’ll close registration during the voting period.
+We’ll use emails from the “election-voter” group members.
+
+## Nominees
+
+During the same period, we invite every community member to nominate candidates by sending a message to the group link:https://community.jenkins.io/g/election-committee[election-committee] mentioning the position and the motivation.
+On the 31 of October, the nomination period is over. We’ll notify every nominees and get confirmation that they are interested in running as a candidate.
+The list of candidates will be announced on the 7th of November.
+
+**Everybody can nominate candidates.**
+
+This year we are looking for nominees for the following positions:
+
+- link:https://www.jenkins.io/project/governance/#governance-board[Board members]
+- link:https://www.jenkins.io/project/team-leads/#documentation[Documentation Officer]
+- link:https://www.jenkins.io/project/team-leads/#events[Events Officer]
+- link:https://www.jenkins.io/project/team-leads/#infrastructure[Infrastructure Officer]
+- link:https://www.jenkins.io/project/team-leads/#release[Release Officer]
+- link:https://www.jenkins.io/project/team-leads/#security[Security Officer]
```

Comment:
Generally prefer site local links so that we can test the links locally while running a local copy
```suggestion
- link:/project/governance/#governance-board[Board members]
- link:/project/team-leads/#documentation[Documentation Officer]
- link:/project/team-leads/#events[Events Officer]
- link:/project/team-leads/#infrastructure[Infrastructure Officer]
- link:/project/team-leads/#release[Release Officer]
- link:/project/team-leads/#security[Security Officer]
```